src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -94,4 +96,14 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+	public static Comparator<ReadOnlyPerson> PersonNameComparator = new Comparator<ReadOnlyPerson>(){
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.